packages/core/src/browser/tree/tree-model.ts

```diff
@@ -410,7 +420,8 @@ export class TreeModelImpl implements TreeModel, SelectionProvider<ReadonlyArray
         this.selectionService.addSelection(selectionOrTreeNode);
     }
 
-    selectNode(node: Readonly<SelectableTreeNode>): void {
+    selectNode(node: Readonly<SelectableTreeNode>, selectedByNavigator?: boolean): void {
```

Comment:
We renamed `onScrollToNodeRequested` but here we still refer to `navigator`.
The `@theia/core` extension should not know about downstream extensions so please generalize.

Alternatively, can we not implement `selectNode` in `navigator-model` to use `center` and not touch `tree-model`.